docs/using-the-jdbc-driver/DataSource.md

```diff
@@ -5,9 +5,9 @@ You can use the `DriverManager` class or a datasource to establish a new connect
 
 To establish a connection with the AwsWrapperDataSource, you must:
 
-1. Configure the property names for the underlying driver-specific datasource. 
-2. Target a driver-specific datasource.
-3. Configure the driver-specific datasource.
+1. Target a driver-specific datasource.
+2. Configure the driver-specific datasource.
+3. Configure any AWS JDBC Driver properties.
```

Comment:
We should be more specific here too...

"Configure any additional properties of the JDBC driver directly in the datasource object"